### PR TITLE
[Integrations] New parameters for How it Works section

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR attempts to add a couple new parameters to the how-it-works section of the single integration page template.

**Notes for Reviewers**

This attempt seems to have failed, though, and could use some help.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f2e9a8d5efd3fcf231f5d0a8c021f8e6e918e284 at: https://6348d02f358daa4e64e33ffa--layer5.netlify.app

**Comment by sahilsuman933** [comment]
You need to add `howItWorks` and `howItWorksDetails` in the GraphQL  query, which can be found in this location `src/templates/integration.js.`

I don't have write access to this repository. Otherwise, I would have made the commit myself.

There are a few things to note before merging this PR: 

1. We need to add How it Works section for all other integrations. Otherwise, it will show a blank screen on the individual integration page in the How it Works section. 

2. There already exists a Kubernetes page. So we don't need another page for it.

// @leecalcote

**Comment by l5io** [comment]
🚀 Preview for commit f61ec1fb9ce6f4772d51f689d1b247a4b045dd29 at: https://63495f9cde277b47fff68f30--layer5.netlify.app

**Review by sahilsuman933 — APPROVED** [review]

**Pull request merged**
